mongomock/aggregate.py

```diff
@@ -625,6 +626,24 @@ def _handle_conditional_operator(self, operator, values):
             'aggregation pipeline, it is currently not implemented '
             ' in Mongomock.' % operator)
 
+    def _handle_control_flow_operator(self, operator, values):
+        if operator == '$switch':
+            branches = values['branches']
+            default = values.get('default')
+            for branch in branches:
+                try:
+                    case = self.parse(branch['case'])
+                except KeyError:
+                    case = False
+                if case:
+                    return self.parse(branch['then'])
+            return self.parse(default)
+        # This should never happen: it is only a safe fallback if something went wrong.
+        raise NotImplementedError(  # pragma: no cover
+            "Although '%s' is a valid conditional operator for the "
```

Comment:
I think you forgot to update the text here: it's not a conditional operator but a control flow operator.